models/solana_utils/solana_utils_daily_balances.sql

```diff
@@ -6,50 +6,48 @@
         materialized='incremental',
         file_format = 'delta',
         incremental_strategy='merge',
+        incremental_predicates = [incremental_predicate('block_time')],
         unique_key = ['token_mint_address', 'address','day'],
         post_hook='{{ expose_spells(\'["solana"]\',
                                     "sector",
                                     "solana_utils",
                                     \'["ilemi"]\') }}')
 }}
 
-WITH 
+WITH
       tokens_accounts as (
-            SELECT 
+            SELECT
             distinct address
-            FROM {{ source('solana','account_activity') }}
-            WHERE tx_success
```

Comment:
@andrewhong5297 can we do this? The diff here is that `tx_success` is not in the where for `solana_utils.token_accounts`

Reply:
errr idk what this ref file is. is this solana_utils.token_accounts? if so then yeah that is probably fine

Reply:
It's this one https://github.com/duneanalytics/spellbook/blob/main/models/solana_utils/solana_utils_token_accounts.sql